docs/teleport/README.md

```diff
@@ -0,0 +1,99 @@
+# Project Teleport Overview (Private Preview)
+
+Instancing a custom environment within seconds is one of the many wonders of running containers. Having to wait for the image and its layers to download & decompress the first time is the current price of admission.
+
+***Project Teleport removes the cost of download and decompression by mounting pre-expanded layers from the Azure Container Registry to Teleport enabled Azure container hosts.***
+
+|Dedicated VM|Teleport w/ACR Tasks |
+|-|-|
+|![](./media/vm-aci.gif)|![](./media/tasks-aci.gif)|
+
+[![](./media/AzureFridayTeleportPreviewThumb.png)](https://channel9.msdn.com/Shows/Azure-Friday/How-to-expedite-container-startup-with-Project-Teleport-and-Azure-Container-Registry/player#time=21s)
+
+![](./media/teleport-metrics.png)
+> For more background, please see [Azure Container Registry Adds Teleportation][teleport-blog-post]
+
+## Table of Contents
+
+- [Sign Up for the Project Teleport Preview](#sign-up-for-the-project-teleport-preview)
+- [Supported Services](#supported-services)
+- [Preview Constraints](#preview-constraints)
+- [Getting Started with Teleportation](./getting-started.md)
+- [Getting Support](#getting-support)
+
+## Sign Up for the Project Teleport Preview
+
+In these early stages, we're looking for direct feedback. To request access, please sign up here: [aka.ms/teleport/signup][signup]
+
+## Supported Services
+
+- ~~Preview 1 focused on running containers within [ACR Tasks][acr-tasks].~~
```

Comment:
I suggest we remove the `preview 1` for ACR Tasks in this doc. Or mention it will be deprecated.

Reply:
All content related to ACR Tasks has been removed in this PR. I didn't want to explicitly say a negative. 
By slashing it out, it's implied we're onto Preview 2, focusing uniquely on AKS.

Reply:
ok